c2corg_ui/static/js/addassociation.js

```diff
@@ -88,7 +88,8 @@ app.AddAssociationController.prototype.associate = function(doc) {
   // if the parent doc is a route and the child doc is a waypoint OR
   // if the parent doc is an outing, inverse the IDs.
   if ((parentType === 'routes' && doc['type'] === 'w') ||
-      parentType === 'outings' || parentType === 'images' ||
+      (parentType === 'outings' && (doc['type'] === 'r' || doc['type'] === 'u')) ||
```

Comment:
Outings were swapped to be child in all added associations on the outing view page (but in the case of articles we should keep them as children in its association to outings and there is no need to include them here - for outings only route/user should be swapped). https://github.com/c2corg/v6_common/blob/master/c2corg_common/associations.py#L7-L8